frontend/components/App.tsx

```diff
@@ -165,36 +106,22 @@ function App() {
       }
     };
     ipcRenderer.on('db-lists', dbListFromBackend);
-    // return cleanup function
     return () => {
       ipcRenderer.removeListener('db-lists', dbListFromBackend);
     };
   });
 
-  /**
-   * New central source of async calls
-   */
+  // Handle async calls
   const asyncCount = useRef(0);
   useEffect(() => {
     const { issued, resolved, asyncList } = menuState.loading;
-    // Check that we are here because a new async was issued
     if (issued - resolved > asyncCount.current) {
```

Comment:
Okay with removing comments so long as we replace/recomment the new functionality to be implemented.